Library/Formula/agda.rb

```diff
@@ -101,11 +131,13 @@ module simple-test where
       snoc [] x = x :: []
       snoc (x :: xs) y = x :: (snoc xs y)
     EOS
-    system bin/"agda", "-c", "--no-main", "--safe", test_file_path
+    if build.with? "malonzo"
+      system bin/"agda", "-c", "--no-main", "--safe", test_file_path
+    end
```

Comment:
I don’t understand your comment. I’ve left the contents of the tests as I’ve found them. Can you clarify?